### DISCUSSION TITLE
Test discission

### DISCUSSION THREAD

**Discussion by GhislainTselone** [Cyfrin/foundry-full-course-cu#1137] [General]
I'm getting this error while compiling my solidity code:

```
///`cid` is an invalid Ipfs-Cid
    /// @param cid Ipfs-Cid entered.
    error InvalidCid(string cid)
```

here  is my code:

```solidity 
        require(msg.sender != address(0), "Cannot be called from 0x0 address.");
```

can someone help me understand it ?

**Comment by vigneshbalu96** [comment]
🚨  could you show some more lights on where you tried to run the code? and are you using IPFS to store & interact with any of the data? On which lesson you are facing the issue or it is a unique your own problem?

🔥 ### **Remember** : we need to work incredibly hard in order to be  incredibly lazy. 🔥

**Comment by GhislainTselone** [comment]
I was just learning to create and publish issues. This is not actually related to a real concern.
Thanks!
